### PR TITLE
Further fixes

### PR DESCRIPTION
**Pull request by josch1710**
Fixed some compilation error in audio part.
Changed tabs to spaces in OS/2 joystick code

### CODE REVIEW

File: src/audio/os2/SDL_os2audio.c [hunk 1]
```diff
@@ -454,7 +454,7 @@ static int OS2_OpenDevice(_THIS, const char *devname)
     }
     pAData->usDeviceId = stMCIAmpOpen.usDeviceID;
 
-    if (iscapture) {
+    if (_this->iscapture) {
```
Comment (sezero) [inline src/audio/os2/SDL_os2audio.c:457]: Why is this necessary? `iscapture` is a local already assigned to `_this->iscapture` .

Comment (SilvanScherrer) [reply src/audio/os2/SDL_os2audio.c:457]: dont worry. we will not merge it

File: src/video/SDL_video.c [hunk 2]
```diff
@@ -127,16 +127,16 @@ static VideoBootStrap *bootstrap[] = {
 #if SDL_VIDEO_DRIVER_QNX
     &QNX_bootstrap,
 #endif
-#if SDL_VIDEO_DRIVER_OFFSCREEN
-    &OFFSCREEN_bootstrap,
-#endif
 #if SDL_VIDEO_DRIVER_NGAGE
```
Comment (sezero) [inline src/video/SDL_video.c:130]: mainstream has ngage _after_ os2

File: src/audio/os2/SDL_os2audio.c (hunk 1 above)
Comment (sezero) [reply src/audio/os2/SDL_os2audio.c:457]: ok then
